src/main/java/com/boomi/flow/services/boomi/mdh/database/FieldMapper.java

```diff
@@ -109,34 +108,84 @@
         return types;
     }
 
-    public static Map<String, Object> createMapFromMobject(MObject mObject) {
+    public static Map<String, Object> createMapFromModelMobject(MObject mObject, Universe universe) {
         Map<String, Object> mapObject = new HashMap<>();
 
         for (Property property: mObject.getProperties()) {
             if (property.getDeveloperName().startsWith("___")) {
                 continue;
+            } else {
+                Object object = createMapEntry(property, universe.getLayout().getModel().getElements());
+
+                if (object != null) {
+                    if (property.getContentType() != ContentType.Object && property.getContentType() != ContentType.List) {
+                        // is not a field group
+                        mapObject.put(property.getDeveloperName(), object);
+                    } else if (property.getContentType() == ContentType.List) {
+                        mapObject.put(getEntryName(property, universe.getLayout().getModel().getElements()), object);
+                    } else if (property.getContentType() == ContentType.Object && object instanceof List) {
```

Comment:
Is there ever an instance where the content type would be `ContentType.Object` and the value would be a single object, or does MDH do the same thing as Flow (i.e. an object is represented as a list with 1 object)?

Reply:
the list is wrapped by an XML tag like is showing this real response test, but not the field group not repeatable 

https://github.com/manywho/service-mdh/pull/16/files#diff-a31f55317875cd6c71f45a56d63606abR7